src/db.c

```diff
@@ -790,7 +790,7 @@ void selectCommand(client *c) {
         return;
     }
     if (selectDb(c, id) == C_ERR) {
-        addReplyError(c, "DB index is out of range");
+        addReplyError(c, "DB dbix is out of range");
     } else {
```

Comment:
Why change this client response?

Reply:
Observed it's been changed in multiple client response(s). I think we can skip these changes.

Reply:
Yeah, not a great find/replace before I got pulled back into meetings 😓. Latest version is updated.